gnome-initial-setup/pages/account/um-photo-dialog.c

```diff
@@ -203,6 +203,10 @@ get_facesdirs (void)
         const char *facesdir_env;
         GisDriver *driver;
 
+        GSettingsSchemaSource *source = g_settings_schema_source_get_default ();
+        g_autoptr(GSettingsSchema) schema = NULL;
+        g_autoptr(GSettings) settings = NULL;
```

Comment:
Small nitpick, I would move this inside the `if (schema)` block so that it is freed earlier when the block ends.

Reply:
This comment no longer makes sense in the updated code.